distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/axis2/axis2.xml.j2

```diff
@@ -572,7 +572,7 @@
         <phase name="StatReporting"/>
     </phaseOrder>
 
-    <clustering class="org.wso2.carbon.core.clustering.hazelcast.HazelcastClusteringAgent"
+    <clustering class="{{clustering.agent}}"

```

Comment:
If you're templating the cluster agent, please add that value to the default.json file
(/distribution/kernel/carbon-home/repository/resources/conf/default.json)
"clustering.agent": "org.wso2.carbon.hazelcast.HazelcastClusteringAgent"